### PR TITLE
Fixing new adopters form height

### PR DESCRIPTION
**Pull request by AmbrishRamachandiran**
## Hey, I just made a Pull Request!

This PR is for https://github.com/backstage/backstage/issues/22123

lastest screen after fix

image

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by npiyush97 — COMMENTED** [review]
translateY(-40%) ?

**Comment by npiyush97** [comment]
can you change the button style appropriately failing a11y test.
https://dequeuniversity.com/rules/axe/4.8/color-contrast?application=AxeChrome

**Comment by AmbrishRamachandiran** [comment]
> can you change the button style appropriately failing a11y test. https://dequeuniversity.com/rules/axe/4.8/color-contrast?application=AxeChrome

can you add more info ? why button style is involved here for fixing height issue?

**Comment by AmbrishRamachandiran** [comment]
Awaiting review commenting for avoid stale

File: microsite/src/pages/home/hubSpotNewAdoptersForm.module.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
   right: 0;
   display: flex;
   position: fixed;
-  transform: translateY(-50%);
+  transform: translateY(-40%);
```
Comment (benjdlambert) [inline microsite/src/pages/home/hubSpotNewAdoptersForm.module.scss:6]: I'm not sure that this a fix for this. This is meant to be half of the size of the element, I think you'll probably want to make sure that the element has a different sizing.

**Review by benjdlambert — CHANGES_REQUESTED** [review]
Don't think that this is the right fix unfortunately. I'm also struggling to reproduce this issue, but I think you will want to make sure that the adopter container flexes and has padding top and bottom in order to make the contents fit correctly.

**Comment by freben** [comment]
@npiyush97 That's a very separate issue though, feel free to file an issue for it (or even contribute a fix if possible).

**Pull request closed without merging**